test/utils/helpers.ts

```diff
@@ -129,8 +122,7 @@ export const vaultStatusToLabel = (status: VaultStatusExt): string => {
 // use the same exchange rate as the running oracles use
 // to avoid flaky tests due to updated prices from the oracle client(s)
 // note: currently the same for all collateral currencies - might change in future
-export const getExchangeRateValueToSetForTesting = <U extends UnitList>(_collateralCurrency: Currency<U>): Big =>
-    new Big("230.0");
+export const getExchangeRateValueToSetForTesting = (_collateralCurrency: Currency): Big => new Big("230.0");
```

Comment:
I think `_collateralCurrency` was only passed for type inference and can now be removed

Reply:
I introduced it back in the day in case we need/want different rates for different currencies.